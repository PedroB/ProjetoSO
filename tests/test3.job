WRITE [(e7,79)(v5,9)(t2,59)(y0,94)]
READ [e7]
WRITE [(l5,27)]
WRITE [(k7,64)(x5,98)]
WRITE [(u9,9)]
READ [t2]
BACKUP
WRITE [(f8,4)(h9,11)]
WRITE [(y6,99)(y2,8)]
WRITE [(w9,40)(b5,98)(z0,60)(i6,38)]
SHOW