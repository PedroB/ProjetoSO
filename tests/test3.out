[(e7,79][(t2,59](a, anna)
(b5, 98)
(b, bernardo)
(d, dinis)
(e7, 79)
(f8, 4)
(h9, 11)
(i6, 38)
(k7, 64)
(l5, 27)
(t2, 59)
(u9, 9)
(v5, 9)
(w9, 40)
(x5, 98)
(y2, 8)
(y6, 99)
(y0, 94)
(z0, 60)
